test/unit/extension/lib/catalog/product/service/ShopgateProductBuilderTest.js

```diff
@@ -75,13 +73,12 @@ describe('Pricing related tests: ', () => {
   })
 
   describe('Price strikken out due to discount', () => {
-
-    it('Should strike out the parent\'s original price as it\'s higher than the original', () => {
+    it("Should strike out the parent's original price as it's higher than the original", () => {
```

Comment:
now you added double quotes here?
edit: I see, you did it because of escape characters. well, fine